### DISCUSSION TITLE
Export par un gestionnaire des chiffres de ses cantines

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1465] [Idées]
**Besoin :**
Un besoin exprimé par une utilisatrice, qui travaille pour une Région : pouvoir exporter sous forme de tableau Excel les chiffres renseignés dans les diagnostics de ses cantines.

**Intérêt :**
Pendant la période de télédéclaration, pouvoir identifier, d'un seul coup d'œil et sans avoir à aller sur la fiche de chacune des cantines, les lycées qui n'ont pas rempli leurs données. Cela lui permettrait ensuite de relancer ceux qui n'ont pas encore rempli leurs données.

**Comment by hfroot** [comment]
Ça relance une ancienne idée :  https://docs.google.com/presentation/d/1bpQ7WgxnJ5Si_HPFBiBkS_d9AKjclxRxIzyGehn2Tus/edit#slide=id.g107427cd226_1_21
Pour un autre format d'affichage des données pour mieux repondre au besoins des gestionnaires de bcp de cantines
On pourrait imaginer un export excel/ods/csv de ça en plus
![Screenshot from 2022-06-08 10-56-44](https://user-images.githubusercontent.com/9282816/172575852-4a5a2309-eaa1-4f12-84d9-af4ddd628146.png)

**Reply by ghost** [reply]
Super.
